docs/machine-learning/tutorials/sentiment-analysis-model-builder.md

```diff
@@ -60,70 +63,119 @@ Each row in the *wikipedia-detox-250-line-data.tsv* dataset represents a differe
+
+## (Optional) Consume the model
 
-If you're not satisfied with your accuracy metrics, some easy ways to try to improve model accuracy are to increase the amount of time to train the model or use more data. Otherwise, select the **code** link to move to the final step in the Model Builder tool.
+This step will have project templates that you can use to consume the model. This step is optional and you can choose the method that best suits your needs on how to serve the model.
 
-## Add the code to make predictions
+- Console App
+- Web API
 
-Two projects will be created as a result of the training process.
+When adding a console app to your solution, you will be prompted to name the project.
 
-### Reference the trained model
+1. Name the console project **SentientAnalysis_Console**.
+1. Click **Add to solution** to add the project to your current solution.
+1. Run the application.
 
-1. In the *code* step of the Model Builder tool, select **Add Projects** to add the autogenerated projects to the solution.
+    The output generated by the program should look similar to the snippet below:
 
-    The following projects should appear in the **Solution Explorer**:
+    ```bash
+    SentimentText: ==RUDE== Dude, you are rude upload that carl picture back, or else.
+    LoggedIn: True
 
-    - *SentimentRazorML.ConsoleApp*: A .NET Core Console application that contains the model training and prediction code.
-    - *SentimentRazorML.Model*: A .NET Standard class library containing the data models that define the schema of input and output model data as well as the saved version of the best performing model during training.
+    Predicted Sentiment: 1    
+    ```
+
+### Web API
+
+When adding a web API to your solution, you will be prompted to name the project.
+
+1. Name the Web API project **SentimentAnalysis_API**.
+1. Click **Add to solution** to add the project to your current solution.
+1. Run the application.
+1. Open PowerShell and enter the following code where PORT is the port your application is listening on.
+
+    ```powershell
+    $body = @{
+        SentimentText="Model Builder is cool!",
+    }
 
-    For this tutorial, only the *SentimentRazorML.Model* project is used because predictions will be made in the *SentimentRazor* web application rather than in the console. Although the *SentimentRazorML.ConsoleApp* won't be used for scoring, it can be used to retrain the model using new data at a later time. Retraining is outside the scope of this tutorial though.
+    Invoke-RestMethod "https://localhost:<PORT>/predict" -Method Post -Body ($body | ConvertTo-Json) -ContentType "application/json"
+    ```
+
+1. If successful, the output should look similar to the text below:
+
+    ```powershell
+    prediction score
+    ---------- -----
+    0 {0.022516366, 0.97748363}
+    ```
 
 ### Configure the PredictionEngine pool
```

Comment:
This section is not part of the **(Optional) Consume the model section**.

Reply:
For this example, I'm thinking we might just get rid of the Consume the model section to avoid confusion.